code-quality-game-docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3"
+
+services:
+
+  code-quality-game:
+    image: maven:3.5.2-jdk-8
+    command: mvn spring-boot:run
+    expose:
+      - "8080"
+    ports:
+      - "8080:8080"
+    volumes:
+      - ~/.m2-docker:/root/.m2
```

Comment:
I'm not sure about this one, why is it needed?

Reply:
hi, sorry for being late, I used to have m2 path inside docker in another path different from the user to not being corrupted (user root or docker writes inside m2 default path and could break your local maven repository)